server/lib/launch-editor.js

```diff
@@ -0,0 +1,356 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file can be found at https://github.com/facebook/create-react-app/blob/next/LICENSE
+ *
+ * This file was modified by Henrik Wenz in order to fit to the next.js project.
+ * This file should be removed as soon as https://github.com/facebook/create-react-app/issues/4899 got resolved.
+ */
+import fs from 'fs'
+import path from 'path'
+import childProcess from 'child_process'
+import os from 'os'
+import chalk from 'chalk'
```

Comment:
Let's remove the chalk dependency and just call `console.log`. This saves us from adding another dependency